content/en/docs/reference/glossary/master.md

```diff
@@ -0,0 +1,15 @@
+---
+title: Master
+id: master
+date: 2020-04-16
+short_description: >
+  Deprecated term, used as synonym for nodes running the control-plane.
+
+aka:
+tags:
+- fundamental
+---
+ Deprecated term, used as synonym for {{< glossary_tooltip text="nodes" term_id="node" >}} hosting the {{< glossary_tooltip text="control plane" term_id="control-plane" >}}.
```

Comment:
Is “master” formally deprecated? I think “control plane” is better for a number of reasons _but_ I don't know that there's a documented deprecation decision for the term “master”.

Reply:
That's an excellent point. I was based on the discussion https://github.com/kubernetes/website/issues/652, but I didn't find anything beside that. The term is still used at https://github.com/kubernetes/community/blob/master/contributors/design-proposals/architecture/architecture.md, for example.